react-props-and-expressions/my-app/src/CustomButton.tsx

```diff
@@ -0,0 +1,8 @@
+type Props = {
+  text: string;
+  color: string;
+};
+
+export function CustomButton({ text, color }: Props) {
+  return <button className={color}>{text}</button>;
```

Comment:
Since your prop is really a class name, I suggest you name it `className`. Either that, or make it an actual color by setting the background color using the `style` prop.